src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig

```diff
@@ -22,4 +22,107 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  *#}
-{% use '@PrestaShop/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig' %}
+{% use 'bootstrap_4_horizontal_layout.html.twig' %}
```

Comment:
As @eternoendless explained in his PR this refers to Symfony bootstrap layout, not the one included in PrestaShop I think it would be worth explaining it through a twig comment here Or people will keep modifying "our" `PrestaShop/Admin/TwigTemplateForm/bootstrap_4_horizontal_layout.html.twig` file

Reply:
Can we do this in another PR ? If we modify this PR content, then it's not a backport anymore

Reply:
Added comment in PR #16938: https://github.com/tdavidsonas88/PrestaShop/blob/11ef472749df2e64aba3e626e3aff0b9fb1aebe8/src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig#L26-L33